Libraries/Notification/NotificationAndroid.js

```diff
@@ -0,0 +1,93 @@
+/**
```

Comment:
Should this be NotificationAndroid.android.js and should we add a .ios.js that is not implemented rather than just marking all the methods below as @platform android?